sdk/purview/azure-analytics-purview-account/README.md

```diff
@@ -0,0 +1,100 @@
+    <version>1.2.5</version>
+</dependency>
+```
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
+
+##### Example
+<!-- embedme ./src/samples/java/com/azure/analytics/purview/account/ReadmeSamples.java#L7-L10 -->
+```java
+AccountsClient client = new PurviewAccountClientBuilder()
+    .host(System.getenv("ACCOUNT_ENDPOINT"))
+    .credential(new DefaultAzureCredentialBuilder().build())
+    .buildAccountsClient();
+```
+
+## Key concepts
+
+## Examples
+More examples can be found in [samples][samples_code].
+
+## Troubleshooting
+
+### Enabling Logging
+
+Azure SDKs for Java offer a consistent logging story to help aid in troubleshooting application errors and expedite
+their resolution. The logs produced will capture the flow of an application before reaching the terminal state to help
+locate the root issue. View the [logging][logging] wiki for guidance about enabling logging.
+
+## Next steps
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights to use your contribution.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
+
+<!-- LINKS -->
+[samples]: src/samples/java/com/azure/analytics/purview/account
+[source_code]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/purview/azure-analytics-purview-account/src
+[samples_code]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/purview/azure-analytics-purview-account/src/samples/
+[azure_subscription]: https://azure.microsoft.com/free/
+[api_reference_doc]: https://azure.github.io/azure-sdk-for-java
+[product_documentation]: https://azure.microsoft.com/services/purview/
+[azure_identity]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/identity/azure-identity
+[DefaultAzureCredential]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/identity/azure-identity/README.md#defaultazurecredential
+[jdk_link]: https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable
+[package]: https://mvnrepository.com/artifact/com.azure/azure-analytics-purview-account
+[samples_readme]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/purview/azure-analytics-purview-account/src/samples/README.md
+[low_level_client]: https://github.com/Azure/azure-sdk-for-java/wiki/Low-Level-Client
```

Comment:
No such page as https://github.com/Azure/azure-sdk-for-java/wiki/Low-Level-Client

Is it https://github.com/Azure/azure-sdk-for-java/wiki/Protocol-Methods ?

Reply:
Fixed. I also need to update the wiki content.

Reply:
Maybe we should not call it "Low-Level client documentation"? @jianghaolu for suggestion on name. "Protocol client documentation"?

Reply:
Fixed